changelog.txt

```diff
@@ -2,6 +2,7 @@ V.Next
 ----------
 - [MINOR] Hide Switch Account in Broker interactive flows (#1284)
 - [MINOR] Enables removeAccount api to remove account records from all environments (#1248)
+- [PATCH] Generic Timeout Rule for tests added (#1286)
```

Comment:
This is automation only change - doesn't even need a changelog entry...we can remove this